Makefile

```diff
@@ -22,7 +22,7 @@ build-tdexdconnect:
 proto: proto-lint
```

Comment:
We should add external dep download cmd's (buf mod update & buf build), also find it useful to run go mod tidy in the end, so my suggestion would be:

```
## proto: compile proto files
proto: proto-lint
	@echo "Compiling stubs..."
	@cd api-spec/protobuf; buf mod update; buf build
	@buf generate buf.build/tdex-network/tdex-protobuf
	@buf generate
	@make install
```